Error after upgrading to React Native 0.72

After upgrading to React Native 0.72 and trying to start the App, I get this error now:

> FAILURE: Build failed with an exception.
> 
> * Where:
> Build file '/repo-template/node_modules/react-native-notifications/lib/android/app/build.gradle' line: 100
> 
> * What went wrong:
> Execution failed for task ':tasks'.
> > Could not create task ':react-native-notifications:testReactNative60DebugUnitTest'.
>    > No signature of method: org.gradle.api.reporting.internal.TaskGeneratedSingleDirectoryReport.enabled() is applicable for argument types: (Boolean) values: [true]

--- a/lib/android/app/build.gradle
+++ b/lib/android/app/build.gradle
@@ -97,7 +97,7 @@ android {
     testOptions {
         unitTests.all { t ->
             reports {
-                html.enabled true
+                html.required.set true
             }
             testLogging {
                 events "PASSED", "SKIPPED", "FAILED", "standardOut", "standardError"

